docs/solidity-to-cadence.mdx

```diff
@@ -0,0 +1,447 @@
+interoperability on Ethereum are designed around addresses (or more specifically the account associated with an
+address), resulting in all contracts having to carefully track and evaluate access and authorizations.
+
+Transactions are based on who authorized them, which is provided as `msg.sender` in the transaction context.
+User to contract, or contract to contract interactions must be explicitly coded to ensure the appropriate approvals
+have been made before interacting with a contract. The contract based nature of storage means that user ownership
+in Ethereum is represented in a mapping, for example from owner to balance, or token ID to owner. Put another
+way, ownership is tracked in ledger records similar to a person's bank balance. Crypto wallets help combine
+balances from multiple token types into a convenient view for the user.
+
+<Img src="https://storage.googleapis.com/flow-resources/documentation-assets/solidity-to-cadence/ethereum-ownership.png" />
+
+Cadence introduces new primitives and distinct functionalities, namely Resources and Capabilities, that are designed
+around Flow's account model. Resources are first-class language types which are unique, non-copyable, and which cannot
+be discarded. These properties make Resources ideal for representing tokens that are always limited in numbers.
+Resources are always stored in account storage and contracts control access to them using Capabilities. Capabilities
+are another special type that secure protected resources without the need for tracking addresses. Cadence makes
+working with these straightforward and intuitive to those familiar with object-oriented programming languages.
+
+Newcomers to Cadence should ensure they understand the following major concepts before development.
+
+## Flow account model
+
+The [Flow account model](learn#concepts#accounts-and-keys) in Cadence combines storage for the keys and code
+(”smart contracts”) associated with an account with storage for the assets owned by that account. That’s right:
+In Cadence, your tokens are stored in your account, and not in a smart contract. Of course, smart contracts still
+define these assets and how they behave, but those assets can be securely stored in a user’s account through the
+magic of Resources.
+
+There is only one account type in Cadence also with an account address, similar to an Externally-Owned-Account
+(EOA) address in Ethereum. However, unlike Ethereum contract-accounts, accounts in Cadence also store contract code.
+Accounts realize ownership on Flow in being the container where keys, Resources, and contracts are stored on-chain.
+
+<Img src="https://storage.googleapis.com/flow-resources/documentation-assets/solidity-to-cadence/account-structure.png" />
+
+### PublicAccount and AuthAccount
+
+Fields, functions, Resources and/or Capabilities in a contract use [access modifiers](language@access-control)
```

Comment:
```suggestion
Fields, functions, Resources and/or Capabilities in a contract use [access modifiers](https://developers.flow.com/cadence/language/access-control)
```

Reply:
I've seen there are more of this so probably you are planning to update all at the same time